.github/workflows/main.yml

```diff
@@ -10,6 +10,26 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  check_changed_folders:
```

Comment:
This would need to be added to the `check` job's `needs` list if this job is really necessary.